src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const palindromeRegExp = /[\W_]/g;
```

Comment:
this regexp is used to extract alphabetic characters, so I'd name it smth like: alphabeticCharsRegExp.